src/lib/isFQDN.js

```diff
@@ -30,6 +30,10 @@ export default function isFQDN(str, options) {
     if (/[\s\u2002-\u200B\u202F\u205F\u3000\uFEFF\uDB40\uDC20]/.test(tld)) {
       return false;
     }
+    if (Array.isArray(options.require_tld)) {
+      // TODO Array.includes though for browser BC does this
```

Comment:
I am very curious about this comment.

Reply:
@ezkemboi yeah I am as well,  comment should be removed.. and includes should be used (next line).. though then IE is no longer supported.